sdk-python/test/get_pricing_test.py

```diff
@@ -1,6 +1,6 @@
 import unittest
 
-from omnilogger import get_pricing
+from omnilogger.get_pricing import get_pricing
```

Comment:
Is this necessary? I feel like the previous looked cleaner

Reply:
It is necessary because get_pricing was not supposed to be exposed by the package... I removed it from the __init__ file to replace it with the most interesting function: the cost calculator